stubs/urllib3/urllib3/connectionpool.pyi

```diff
@@ -1,9 +1,16 @@
-from typing import Any
+from _typeshed import Self
+from collections.abc import Mapping
+from logging import Logger
+from queue import Queue
```

Comment:
The fact that there are two `queue` modules here is quite confusing. I think it would be more readable if we just did

```python
import queue
import urllib3.util.queue
```

And then referred to `LifoQueue` as `urllib3.util.queue.LifoQueue`, and `Queue` as `queue.Queue`. It's more verbose, but it's a lot less confusing, in my opinion.

Reply:
That's fixed because that should have been `LifoQueue` in both locations after reviewing 

https://github.com/urllib3/urllib3/blob/1.26.x/src/urllib3/connectionpool.py